static/test_io_close.py

```
@@ -0,0 +1,22 @@
+# _*_ coding:utf-8 _*_
+# !/usr/bin/env python
+
+import html, io, json, os, pickle, random, re, sqlite3, string, sys, subprocess, time, traceback, urllib.parse, \
```

Ruff (0.11.9)

`html` imported but unused

Remove unused import

(F401)